### PR TITLE
Updated README.md (WIP)

### PR DESCRIPTION
**Pull request by AshleyEM**
**Changes**

- Added header anchors for each section.
- Reorganized sections, changed formatting for readability/quick scanning.
- Added instructions from MANUAL-INSTALL.md and MANUAL-UNINSTALL.md to keep everything together. 

The link to the wiki for installation details (https://github.com/kyleneideck/BackgroundMusic/wiki/Installation) doesn't appear to have anything yet, but I kept it in for future contributions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,80 +7,100 @@
...
-## Download
+# Overview
 
-### Version 0.3.1
+**Background Music** gives you control over multiple sources of audio on your desktop without navigating each individual application. It provides the following functionalities:
 
-<a href="https://github.com/kyleneideck/BackgroundMusic/releases/download/v0.3.1/BackgroundMusic-0.3.1.pkg"><img 
-src="Images/README/pkg-icon.png" width="32" height="32" align="absmiddle" />
-BackgroundMusic-0.3.1.pkg</a> (571 KB)
++ Automatically pause/unpause your music player when another audio source is playing/stopped
```
[the remark this answers is not in the document]
Comment (AshleyEM) [reply README.md]: I think your suggestion is better, I've changed it.

File: README.md [hunk 2]
```diff
@@ -7,159 +7,177 @@
...
-Uses "gzcat - | tar x" instead of "tar xz" because gzcat will also check the file's integrity (gzip files
-include a checksum), which makes sure we can't run a half-downloaded copy of build_and_install.sh.
--->
-```shell
-(set -eo pipefail; URL='https://github.com/kyleneideck/BackgroundMusic/archive/master.tar.gz'; \
-    cd $(mktemp -d); echo Downloading $URL to $(pwd); curl -qfL# $URL | gzcat - | tar x && \
-    /bin/bash BackgroundMusic-master/build_and_install.sh -w && rm -rf BackgroundMusic-master)
-```
+1. Open **Terminal**.
+2. To locate `uninstall.sh`, run:`cd /Applications/Background\ Music.app/Contents/Re s/`.
```
Comment (kyleneideck) [inline README.md]: I think the command is missing some chars at the end:
`cd /Applications/Background\ Music.app/Contents/Resources/`

**Comment by kyleneideck** [comment]
I just noticed one more small thing. (Commented above.) Looks good to me otherwise.

File: README.md (hunk 2 above)
Comment (AshleyEM) [reply README.md]: Thanks for pointing that out, I've fixed it.

**Comment by kyleneideck** [comment]
Merged. Thanks again.

**Pull request merged**